qtpy/QtWidgets.py

```diff
@@ -22,6 +22,110 @@
 from qtpy import PythonQtError
 
 
+def patch_qcombobox():
+    """
```

Comment:
Since the function has to be on top it makes the module layout a bit strange. Maybe better than `patch*` would be a submodule `_patch/` so that it doesn't pollute the main directory.